osgp/protocol-adapter-mqtt/osgp-protocol-simulator-mqtt/src/main/java/org/opensmartgridplatform/simulator/protocol/mqtt/SimulatorBrokerConfig.java

```diff
@@ -56,4 +58,9 @@ public Properties mqttBrokerProperties() {
     }
     return p;
   }
+
+  @Bean(initMethod = "start", destroyMethod = "stop")
+  public Broker broker(final Properties mqttBrokerProperties) throws IOException {
```

Comment:
Removed throws clause, as it is no longer required to be there.
I guess the method can still throw an `IOException` though. The explicit call to `Broker.start` that used to be there has been removed, yet `start` is still called because it is specified as `initMethod` for this `Bean`.